[@builder.io/plugin-smartling] add a plugin for translating content with smartling, add translation helpers to utils package (#1090)
BuilderIO/builder: plugins/smartling/README.md

# Builder.io Smartling Integration Plugin

## Installation

From the integrations tab, pick `smartling`

## Translating content

> What's beging traanslated:

Reviewer: ```suggestion
What's being translated:
```

- All text elements in builder content [you can exclude specific element by right click + `exclude from future translations`]

> - All custom fields in content that's marked as `localized`

Reviewer: ```suggestion
- All custom fields in content that are marked as `localized`
```

From what language?
The source language should match your smartling project source locale.

To what languages?
By default it'll request translations for all target locales configured in your project locales, you can use the targeting UI to limit the translation request to a subset of those locales.

How To translate?

> - prepare translation jobs by adding contents to draft jobs by pressign translate in content options or content list options [todo screenshots]

Reviewer: ```suggestion
- prepare translation jobs by adding contents to draft jobs by pressing translate in content options or content list options [todo screenshots]
```
